src/utils/axiosAPI/api.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios'
+
+const baseURL = 'https://api.nasa.gov/'
+const apiKEY = 'api_key=aCNwLvVe8gykdUxPUggWkOt3lB3RzzHoVHTy4ZQZ'
```

Comment:
constants should be UPPERCASED:
**BASE_URL** and **AP_KEY**
have a look at the **airbnb** styleguide, there's a section in there describing when a constant should be UPPERCASED and when not